src/main/java/com/google/sps/servlets/userApi.java

```diff
@@ -23,14 +23,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     JsonObject loginInfo = new JsonObject();
     UserService userService = UserServiceFactory.getUserService();
     Boolean isLoggedIn = userService.isUserLoggedIn();
+
     String url =
         isLoggedIn
-            ? userService.createLoginURL("/index.html")
-            : userService.createLogoutURL("/index.html");
+            ? userService.createLogoutURL("/index.html")
+            : userService.createLoginURL("/index.html");
 
     loginInfo.addProperty("Url", url);
```

Comment:
it might be simpler if you return a json object with either a login_url or logout_url property.

This way your client can use the correct url without having to also check the isLoggedIn boolean.